src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -855,7 +855,7 @@ public unsafe NativeWindow(NativeWindowSettings settings)
             }
             else
             {
-                WindowPtr = GLFW.CreateWindow(settings.Size.X, settings.Size.Y, _title, null, (Window*)(settings.SharedContext?.WindowPtr ?? IntPtr.Zero));
+                WindowPtr = GLFW.CreateWindow(settings.ClientSize.X, settings.ClientSize.Y, _title, null, (Window*)(settings.SharedContext?.WindowPtr ?? IntPtr.Zero));
```

Comment:
It could be that glfw is inconsistent about these these sizes on different platforms, as we discovered in #1651, I can look into this if you don't have the means.

Reply:
I have no way to test for Mac behavior.

Reply:
I can test mac if you want to test windows and linux.

Reply:
I also just let this run on Ubuntu 23.04 X11
and (800, 600) was reported on the initial resize event and in the properties dump

also tested it on Ubuntu 23.10 Wayland and the same there

further made sure Size and ClientSize reported the correct values,
so Size is the ClientSize + border which also seems correct on both

Reply:
I haven't been able to test on macos yet, but if it turns out to be a problem we can fix that in a later version.